src/ErrorHandler.php

```diff
@@ -227,30 +226,23 @@ public function handleFatalError(array $error = null): void
 
         if (!empty($error) && $error['type'] & (E_ERROR | E_PARSE | E_CORE_ERROR | E_CORE_WARNING | E_COMPILE_ERROR | E_COMPILE_WARNING)) {
             $errorAsException = new \ErrorException(self::ERROR_LEVELS_DESCRIPTION[$error['type']] . ': ' . $error['message'], 0, $error['type'], $error['file'], $error['line']);
-        }
-
-        try {
-            if (null !== $errorAsException) {
-                $this->handleException($errorAsException);
-            }
-        } catch (\Throwable $errorAsException) {
-            // Ignore this re-throw
+            $this->invokeErrorListeners($errorAsException);
         }
     }
 
     /**
-     * Handles the given exception by capturing it through the Raven client and
+     * Handles the given exception by passing it to all the listeners,
      * then forwarding it to another handler.
      *
      * @param \Throwable $exception The exception to handle
      *
      * @throws \Throwable
      *
-     * @internal
+     * @internal This method is public only because it's used with set_exception_handler
```

Comment:
I think it's out of scope here, refactoring this would require a big effort: the test is verifying that this method rethrows the exception, so doing that in a test without direct access to the method is complicated.

I would leave this out of this PR.